src/fsharp/FSComp.txt

```diff
@@ -1471,13 +1471,13 @@ keywordDescriptionUntypedQuotation,"Delimits a untyped code quotation."
 3216,itemNotFoundDuringDynamicCodeGen,"%s '%s' not found in assembly '%s'. A possible cause may be a version incompatibility. You may need to explicitly reference the correct version of this assembly to allow all referenced components to use the correct version."
 3216,itemNotFoundInTypeDuringDynamicCodeGen,"%s '%s' not found in type '%s' from assembly '%s'. A possible cause may be a version incompatibility. You may need to explicitly reference the correct version of this assembly to allow all referenced components to use the correct version."
 descriptionWordIs,"is"
-notAFunction,"This value is not a function and cannot be applied."
-notAFunctionButMaybeIndexerWithName,"This value is not a function and cannot be applied. Did you intend to access the indexer via '%s.[index]'?"
-notAFunctionButMaybeIndexer,"This expression is not a function and cannot be applied. Did you intend to access the indexer via 'expr.[index]'?"
-notAFunctionButMaybeIndexerWithName2,"This value is not a function and cannot be applied. Did you intend to access the indexer via '%s[index]'?"
-notAFunctionButMaybeIndexer2,"This expression is not a function and cannot be applied. Did you intend to access the indexer via 'expr[index]'?"
+notAFunction,"This value is not a function, it is a %s. It doesn't take any arguments, but you're trying to pass a value as an argument to it. Consider removing the extra argument."
+notAFunctionButMaybeIndexerWithName,"This value is not a function, it is a %s. It doesn't take any arguments, but you're trying to pass a value as an argument to it. Did you intend to access the indexer via '%s.[index]'?"
+notAFunctionButMaybeIndexer,"This is a expression of type '%s', and not a function. It doesn't take any arguments, but you're trying to pass a value as an argument to it. Did you intend to access the indexer via 'expr.[index]'?"
+notAFunctionButMaybeIndexerWithName2,"This value is not a function, it is a %s. It doesn't take any arguments, but you're trying to pass a value as an argument to it. Did you intend to access the indexer via '%s[index]'?"
+notAFunctionButMaybeIndexer2,"This value is not a function, it is a %s. It doesn't take any arguments, but you're trying to pass a value as an argument to it. Did you intend to access the indexer via 'expr[index]'?"
```

Comment:
@KevinRansom  @vzarytovskii Can you take a look at these messages please, do you like the wording?

Reply:
quick note - while we're changing these messages should the indexer ones suggest dot-less indexing instead?